### ISSUE TITLE
Add Event Widgets to GA Dashboards

### ISSUE THREAD

**Issue by loripusey** [department-of-veterans-affairs/va.gov-team#47812]
## Tasks
- Add Event Widgets to GA Dashboards (Pre-Check-in & eCheck-in)

**Comment by benbrasso-agile6** [comment]
https://app.zenhub.com/workspaces/check-in-experience-61fc23a2cb8a14001132e102/issues/department-of-veterans-affairs/va.gov-team/46171

**Comment by benbrasso-agile6** [comment]
@loripusey I got a bit sidetracked on this task. Wound up creating a new GA Dashboard to more easily communicate the user flow. 

https://analytics.google.com/analytics/web/#/dashboard/WXPApEi8RmGvA8t5236Zzw/a50123418w177519031p176188361/_u.dateOption=last7days/

I'm requesting to put this ticket back in the backlog, and if/when I pick it back up, I'll get back to including more "event widgets" into the other two GA Dashboards. I went this route because I was looking at the "No" responses to the pre-reg questions and sort of got stuck on those for a moment. I also feel like, if I put more time towards this ticket now to finish it, it may risk getting other tickets done. Of course, if I have time left in this sprint to finish adding other "event widgets," I can always pick this back up. 

Thoughts?
